spec/default_facts.yml

```diff
@@ -12,3 +12,4 @@ concat_basedir: "/tmp"
 ipaddress: "172.16.254.254"
 is_pe: false
 macaddress: "AA:AA:AA:AA:AA:AA"
+root_home: /root
```

Comment:
Add this to `default_module_facts.yaml` as well

Reply:
Done

Reply:
If it's in ```default_module_facts.yaml``` then can't it be removed here? This file is modulesynced while the other isn't.

Reply:
Then I'm not that clear of mind yet. Just leave it in and if it's an issue in the modulesync we'll fix it then.